src/util/log.d

```diff
@@ -185,11 +212,11 @@ struct Log
 
 // Helper to allow using named references in format strings, ie. treat %{name} equivalent to %s.
```

Comment:
What's intended for this idea in the future? Will it be deprecated?
Or does it have to be extended like `$foo(42)`?

Reply:
I was intending for it to correspond to named arguments eventually; I think that's still useful. I'd leave it for now. We need more experience with how interpolated strings and named args look in practice.